rfc6962-bis.xml

```diff
@@ -510,186 +596,170 @@ Section 8.2 of <xref target="RFC3447"/>) using a key of at least 2048 bits.
         opaque sct_extension_data&lt;0..2^16-1&gt;;
     } SctExtension;
 
-    SctExtension SctExtensions&lt;0..2^16-1&gt;;
-
     struct {
-        Version sct_version;
         LogID log_id;
         uint64 timestamp;
-        SctExtensions extensions;
+        SctExtension sct_extensions&lt;0..2^16-1&gt;;
         digitally-signed struct {
-            Version sct_version;
-            SignatureType signature_type = certificate_timestamp;
-            uint64 timestamp;
-            LogEntryType entry_type;
-            select(entry_type) {
-                case x509_entry: CertInfo;
-                case precert_entry_V2: CertInfo;
-            } signed_entry;
-            SctExtensions extensions;
-        };
-    } SignedCertificateTimestamp;</artwork>
+            TransItem timestamped_entry;
+        } signature;
+    } SignedCertificateTimestampDataV2;</artwork>
         </figure>
         <t>
-          <spanx style="verb">issuer_key_hash</spanx> is the HASH of the certificate issuer's public key, calculated over the DER encoding of the key represented as SubjectPublicKeyInfo. This is needed to bind the issuer to the final certificate, making it impossible for the SCT to be valid for any other certificate.
+          <spanx style="verb">log_id</spanx> is this log's unique ID, encoded in an opaque vector as described in <xref target="log_id"/>.
         </t>
         <t>
-          <spanx style="verb">tbs_certificate</spanx> is the DER-encoded
-TBSCertificate component of the precertificate. Note that it is also possible to
-reconstruct this TBSCertificate from the issued certificate by extracting the
-TBSCertificate from it, redacting the domain name labels indicated by the
-redacted labels extension, and deleting the SCT list extension and redacted
-labels extension.
+          <spanx style="verb">timestamp</spanx> matches the timestamp from the <spanx style="verb">TimestampedCertificateEntryDataV2</spanx> structure encapsulated in the <spanx style="verb">timestamped_entry</spanx>.
```

Comment:
Addressed in https://github.com/robstradling/certificate-transparency-rfcs/commit/c42b1ea0f759c380708e5d0886dc726c127141a8